### PR TITLE
DBZ-2214 Drop distirbution from distro name

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-2214

### CODE REVIEW

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ NOTE: The above links refers to the nightly snapshot build of the {prodname} mas
 If you are looking for a non-snapshot version, please select the appropriate version of this documentation in the menu to the right.
 endif::[]
 ifeval::['{page-version}' != 'master']
-* https://repo1.maven.org/maven2/io/debezium/debezium-server-distro/{debezium-version}/debezium-server-distro-{debezium-version}-distribution.tar.gz[{prodname} Server distribution]
+* https://repo1.maven.org/maven2/io/debezium/debezium-server-distro/{debezium-version}/debezium-server-distro-{debezium-version}.tar.gz[{prodname} Server distribution]
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:34]: Ok, but didn't we want to remove "distro" also from the artifact id? I.e. https://repo1.maven.org/maven2/io/debezium/debezium-server/{debezium-version}/debezium-server-{debezium-version}.tar.gz

Comment (jpechane) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc:34]: That would mean moving the assmebly at the level of parent project which i found impossible to get running so I had to introduce the distro module in the first place. And as you can see wildlfy has also `dist` in the artifact name https://mvnrepository.com/artifact/org.wildfly/wildfly-dist

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc:34]: Oook. Note that's why I suggested to rename the modules; but let's follow WildFly's example then. "distro" -> "dist"?

Comment (jpechane) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc]: @gunnarmorling Done

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc]: Sweet, thx!

**Pull request merged**